x-pack/test/api_integration/apis/management/rollup/rollup.js

```diff
@@ -45,13 +45,21 @@ export default function ({ getService }) {
 
         const { body } = await supertest.get(uri).expect(200);
 
-        expect(body).to.eql({
-          dateFields: ['testCreatedField'],
-          keywordFields: ['testTagField'],
-          numericFields: ['testTotalField'],
-          doesMatchIndices: true,
-          doesMatchRollupIndices: false,
-        });
+        expect(Object.keys(body)).to.eql([
+          'doesMatchIndices',
+          'doesMatchRollupIndices',
+          'dateFields',
+          'numericFields',
+          'keywordFields',
+        ]);
+
+        expect(body.doesMatchIndices).to.be(true);
+        expect(body.doesMatchRollupIndices).to.be(false);
+        expect(body.dateFields).to.eql(['testCreatedField']);
+        expect(body.keywordFields).to.eql(['testTagField']);
```

Comment:
Can you add an assertion for `numericFields` too?

```js
expect(body.numericFields).to.eql(['testTotalField']);
```

Reply:
Good catch, fixed here https://github.com/elastic/kibana/pull/82547/commits/1ccbe40c1325fbdb15cdf924477eb8c68715317d